.snapcraft/resources/initcaddy

```diff
@@ -1,3 +1,40 @@
-#!/bin/sh
-cp $SNAP/bin/Caddyfile  $SNAP_DATA/Caddyfile
-echo "Replace $SNAP_DATA/Caddyfile with your own to customize reverse proxy" 
+#!/bin/bash
+
+# Config options for Caddyfile
+options="protocol siteurl port"
+
+create_caddyfile(){
+# Copy template to config Caddyfile
+cp $SNAP/bin/Caddyfile $SNAP_DATA/Caddyfile
+
+}
+
+update_caddyfile(){
+
+# Config file path for Caddyfile
+Caddyfile=$SNAP_DATA/Caddyfile
+
+# replace an option inside the config file. 
+refresh_opt_in_config() {
```

Comment:
Just checking.. is this supposed to be nested inside the other function?  It'd potentially read a bit better if it weren't

Reply:
True, I change it